lib/corpus.py

```diff
@@ -163,7 +163,7 @@ def segments(self) -> Iterable[Segment]:
         """
         for r in self.recordings:
             yield from r.segments
-        for sc in self.subcorpora:
+    for sc in self.subcorpora:
```

Comment:
```suggestion
        for sc in self.subcorpora:
```
this change would be a typo, right?

Reply:
Yes, already broke the pipeline twice and already fixed :)